### DISCUSSION TITLE
Mulitple tables with same Model

### DISCUSSION THREAD

**Discussion by MaheshkumarSundaram** [neonexus/sails-react-bootstrap-webpack#102] [General]
Hi @neonexus 

I would like to calrify the below about Sails Model.

I have a model with a schema like:

```js

    attributes: {
        id: {
            type: 'string',
            columnType: 'varchar(36)',
            required: true,
        },

        name: {
            type: 'string',
            required: true,
            maxLength: 70,
        },

        content: {
            type: 'string',
            required: true,
        },

        user: {
            collection: 'user',
            via: 'email'
        },
    },
```
Now, the schema & model is the same for like 6 tables. How to efficiently make use of a single sails model and dynamically change the tables while working?

Any help would be appreciated. Thanks!

**Comment by neonexus** [comment]
Your best bet, would be to make 5 sub-models that all copy from the master model. Since every model is just a Node module, you can build up your attributes in the export.

In other words, `const masterModel = require('myMasterModel.js');`, then `attributes: masterModel.attributes`. Make sense?

**Reply by MaheshkumarSundaram** [reply]
@neonexus 

Yeah. Clearly. Thank you
